packages/react-instantsearch-hooks-dom/src/ui/CurrentRefinements.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+import { isModifierClick } from './lib/isModifierClick';
+
+import type {
+  CurrentRefinementsConnectorParamsItem,
+  CurrentRefinementsConnectorParamsRefinement,
+} from 'instantsearch.js/es/connectors/current-refinements/connectCurrentRefinements';
```

Comment:
it's ok to have imported types from InstantSearch here, but once we implement InstantSearch.js ui components, we'll likely retype these, to avoid circular imports

Reply:
At some point we could have an `instantsearch-core` package with connectors and common types. InstantSearch.js should likely be a flavor like the others, and consume the primitives from a core package. WDYT?

Reply:
yes, that's indeed another option we can consider when ui gets split out 👍